client/modules/wj-css/README.md

```diff
@@ -1 +1,16 @@
+# wj-css
+
 This package follows the guidelines given by [sass-guidelin.es](https://sass-guidelin.es).
+
+> ### NOTE:
+> As it currently stands there are asset import issues - the gist is that you need to
+> be careful if you're using `url()`. If you use it, make sure to use it in a `.css` file.
+> Make sure to import that `.css` file from `wj-css/src/...` so that Vite can resolve
+> where it is. Otherwise, it will try a relative import that will fail.
+>
+> See [`font-faces.css`](src/font-faces.css) and [`main.css`](src/main.scss) for an
+> example of this workaround.
+>
+> This will be fixed at some point - but that'll probably require either compiling
+> this package (rather than letting Vite do it) or upstream fixes from Vite.
+> Currently, only JS imports (e.g. `import "wj-css"`) correctly resolves `url()` paths.
```

Comment:
Just to confirm my understanding - if I need to import something via `url()` rather than by `import`, that needs to be done in a CSS file, as it'll cause resolution issues if done in an SCSS file?

Reply:
Yes, unfortunately. Resolution works _perfectly fine_ if you dynamically import from JS (`import "wj-css"`) but doesn't if you import it literally any other way, it seems. Sass leaves `.css` imports alone, so when Vite sees the `wj-css/src/font-faces.css` import it correctly resolves it because... reasons. I think this is a noted issue upstream and it'll probably get fixed soon.